util/peerKey.go

```diff
@@ -7,15 +7,21 @@ import (
 	crypto "github.com/libp2p/go-libp2p-core/crypto"
 )
 
-//GenEd25519Key Generate a random ED29919, 2048 key
-func GenEd25519Key() (crypto.PrivKey, error) {
+//GenEd25519Key Generate a random ED29919
+func MakeEd25519PeerKey() (string, error) {
```

Comment:
In P2P module , there is a func GenRandPrvKey() , should we make them a util ?

Reply:
The function `GenRandPrvKey` is redundant. We actually do not take any advantage of that wrap. When you use that function, you need to check error again. It basically a duplicated error checking.

Reply:
Agree. Another point is that should we move function identity.go in P2P into util/peerKey.go

Reply:
Not sure. What is `identity.go` making for?

Reply:
Generate random key and decode/encode Hex Key from/to files